### DISCUSSION TITLE
Getting an error when trying to create a policy

### DISCUSSION THREAD

**Discussion by nkapetanas** [eclipse-edc/Samples#252] [Q&A]
Hi!,

I am trying to run the steps in https://github.com/eclipse-edc/Samples/blob/main/transfer/transfer-01-negotiation/README.md and I get an error when trying to run "curl -d @transfer/transfer-01-negotiation/resources/create-policy.json \
  -H 'content-type: application/json' http://localhost:19193/management/v2/policydefinitions \
  -s | jq ". I use cygwin since I am using Windows to run the commands. The issue on the conntector says: 

![image](https://github.com/eclipse-edc/Connector/assets/16067663/9344329a-dc33-4fc1-ad78-a2c368fd5e48)

and in the command line where I run the curl command I am getting: 

![image](https://github.com/eclipse-edc/Connector/assets/16067663/e44763da-32c9-40e2-85a7-9ea12d6cdb47)

Any idea how I can resolve that issue? 

Thank you for your time :)

**Comment by ndr-brt** [comment]
~~please try to replace the policy context with the `https` one: `https://www.w3.org/ns/odrl.jsonld`~~
EDIT: which OS are you on?
